tests/test_legacy_version.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+from message_ix import Scenario
+
+msg_args = ('canning problem (MESSAGE scheme)', 'standard')
+msg_multiyear_args = ('canning problem (MESSAGE scheme)', 'multi-year')
+
+
+def test_solve_legacy_scenario(test_legacy_mp):
```

Comment:
I like the idea, maybe it's worth starting now a naming convention for which version is being tested? Like `test_solve_v1.1_scenario` and `test_v1.1_mp`?

Reply:
Given that future versions will have a hierarchical evolution, I don't think that we will need to add a specific check for each version. Internally, the legacy-db (based on `ixmp 0.1`) will be forwarded to version 0.2, 0.3, etc. sequentially anyway.